app/views/layouts/application.html.haml

```diff
@@ -17,53 +17,66 @@
   %body
     #skiplink-container
       %div
-        %a{ href: '#content', class: 'skiplink' }/
-
-    #beta-banner
-      .beta-wrapper
-        .beta-text
-          BETA
-        .beta-message
-          Please give us your feedback so we can improve this site
+        %a{ href: '#content', class: 'skiplink' }
+          Skip to content
 
     %header#global-header.with-proposition{ role: 'banner' }
       .header-wrapper
         .header-global
           .header-logo
-            %a#logo.content{ href: 'https://crowncommercialservice.gov.uk/', title: 'Go to the Crown Commercial Service homepage' }
-              %img{ src: asset_path('ccs-logo-white.png'), width: 160, height: 131, alt: 'Crown Commercial Service' }
+            %a#logo.content{ href: 'https://www.crowncommercial.gov.uk/', title: 'Go to the Crown Commercial Service homepage' }
+              %img{ src: asset_path('ccs-logo-white.png'), width: 120, height: 100, alt: 'Crown Commercial Service' }
 
         .header-proposition
           .content
-            = render 'shared/top_navigation'
-
-    %main{ role: 'main' }
+            %nav#proposition-menu
+              %a#proposition-name{ href: '/' } Data Submission Service
+              %ul#proposition-links
+                %li
+                  %a{ class: 'active', href: '/' }
+                    Home
+                %li
+                  %a{ href: tasks_path }
+                    Tasks
+                %li
+                  %a{ href: tasks_path }
+                    Account
+                %li
+                  - if current_user
+                    = link_to 'Sign out', '/sign_out'
+                  - else
+                    = link_to 'Sign in', '/auth/auth0'
```

Comment:
In `shared/top_navigation` I've also included the email address in the sign out link. This may not be the best place / not in the designs, but useful to know WHO you're signed in as.

Particularly when we're testing and logging into to multiple accounts at once!